test/integration/taxon_browsing_test.rb

```diff
@@ -20,6 +21,7 @@ class TaxonBrowsingTest < ActionDispatch::IntegrationTest
     and_i_can_see_the_research_and_statistics_section
     and_i_can_see_the_organisations_section
     and_i_can_see_the_sub_topics_grid
+    and_i_can_see_the_in_page_nav
```

Comment:
I would add this in a new test, e.g: `it 'renders an in-page nav in Variant C`. Otherwise it becomes difficult to see what has been added for normal taxon page changes and what has been added for variants.

Reply:
👍